requirements.txt

```diff
@@ -1,29 +1,46 @@
-aiohttp==3.6.2
+aiohttp==3.7.4.post0
```

Comment:
did you run into any issues running the bot after all these upgrades? i'm ok with this, but if we run into any issues running automata on the server with other unrelated plugins breaking we might need to revert this